src/murmur/MurmurIce.cpp

```diff
@@ -40,15 +40,39 @@ void IceStop() {
 	mi = NULL;
 }
 
+/// Remove all NUL bytes from |s|.
+static std::string iceRemoveNul(std::string s) {
+	std::vector<char> newstr;
```

Comment:
Sadly, we can only set the size in the constructor, not the capacity.

And we can only "fit to size" in C++11. :(